modules/ROOT/pages/osv2-faq.adoc

```diff
@@ -46,11 +46,10 @@ Yes, Object Store v2 is in the same region as the worker where the app is initia
 
 IMPORTANT: If after the first deploy, you move the app to a different region, Object Store v2 remains in the original region to avoid data loss. Your use of Object Store v2 never moves from one region to another. When you need to deploy both an app and object store to a different region, delete the app and re-upload the app into the new region. This action loses all the data that may exist in the storage.
 
-== How are Object Store entries divided among workers?
+== Does Object Store v2 support distributed locking for key updates?
```

Comment:
@hannanelson since OSv2 technically *cannot* support distributed locking, we should probably change this to something like
 - Synchronizing access to OSv2

Reply:
@schinthakindims - Since it's no longer a question, I moved it to the OSv2 intro topic.